### ISSUE TITLE
Decide on OWD Q3 projects

### ISSUE THREAD

**Issue by Elchi3** [openwebdocs/project#42]
We met last Wednesday (June 9th, 2021) to go through current OWD project proposals and ideas. 
This issue summarizes them so we can discuss which projects to take on in Q3 2021 (the months July, August, September).

# Proposed projects
Proposed projects are defined ideas that are clear in scope. They are ready to be worked on.
- [Markdown conversation for more of MDN (CSS docs)](https://github.com/openwebdocs/project/issues/25)
- [Finish mixin work (for workers)](https://github.com/openwebdocs/project/issues/23)
- [WebXR Browser Compatibility](https://github.com/openwebdocs/project/issues/35)
- [Fix how MDN distributes Polyfills](https://github.com/openwebdocs/project/issues/27)
- [WebView information on MDN](https://github.com/mdn/content/blob/main/rfcs/webview-information.md)
- [ARIA roles reference docs](https://github.com/mdn/content/blob/main/rfcs/aria-roles.md)
- [Modernizing the Learning Area JavaScript modules](https://github.com/mdn/content/blob/main/rfcs/modernize-learn-js.md)

# Ideas

These ideas aren't proposals yet as more details are needed.
- [Documenting WebDriver](https://github.com/mdn/content/issues/4721) (needs scope, rough outline)
- [Web Testing strategies](https://github.com/mdn/content/issues/3917) (needs scope, rough outline)
- Replace mdn/data with w3c/webref (needs issue / project description)
- [Documenting HTTP/2 and HTTP/3](https://github.com/mdn/content/discussions/5954) (needs issue, rough outline)
- [Fixing MDN page titles](https://github.com/mdn/content/discussions/5121) (needs decision)
- [Fixing how MDN documents dictionaries](https://github.com/mdn/content/discussions/5828) (needs decision, issue, list of pages affected)

# Submit ideas

You can still submit ideas until the end of June and we will take a look!

**Comment by foolip** [comment]
> Finish mixin work (for workers)

Should this also include events? See https://github.com/mdn/browser-compat-data/issues/8929#issuecomment-859545693 for everything that remains to be done to eliminate mixins.

**Comment by Elchi3** [comment]
Yes, ideally, we fix all issues related to mixins.

**Comment by foolip** [comment]
What's the right place to discuss WebView? @tomayac was enthusiastic about WebView, but there's no issue for it, just https://github.com/mdn/content/pull/3918 which added the linked RFC.

**Comment by tomayac** [comment]
"Enthusiastic" as in "good to have this documented". The best browser still is an actual browser, and not a WebView :-)  Just to set this straight.

**Comment by foolip** [comment]
Sorry to overstate your love of WebViews :)

**Comment by Elchi3** [comment]
> What's the right place to discuss WebView? @tomayac was enthusiastic about WebView, but there's no issue for it, just [mdn/content#3918](https://github.com/mdn/content/pull/3918) which added the linked RFC.

I think we should open a new issue here so we can discuss this project some mor, outline a plan, and scope the work that we want to do. 

(As soon as the vacations are over and new OWD employees have been onboarded, I plan to go over all projects, too, but it doesn't hurt to get into some of this stuff beforehand already).

**Comment by Elchi3** [comment]
Forgot to post this here: Results from the stack ranking we did in June:

1) [WebXR Browser Compatibility](https://github.com/openwebdocs/project/issues/35)
2) [Fix how MDN distributes Polyfills](https://github.com/openwebdocs/project/issues/27)
3) [Documenting HTTP/2 and HTTP/3](https://github.com/openwebdocs/project/issues/43)
4) [WebView information on MDN](https://github.com/mdn/content/blob/main/rfcs/webview-information.md)
5) [Modernizing the Learning Area JavaScript modules](https://github.com/mdn/content/blob/main/rfcs/modernize-learn-js.md)
6) [Use webref as the source for CSS formal syntax, instead of mdn/data](https://github.com/openwebdocs/project/issues/44)
7) [ARIA roles reference docs](https://github.com/mdn/content/blob/main/rfcs/aria-roles.md)

We talked about this ranking in the last SC call and I did not hear any reservations, so this order should be a guiding principle when assigning work in our team. We're planning to onboard two new folks soon and we will then get together and look at these to determine what is achievable in the quarter. In the meantime, we're trying to finish continued projects.

**Comment by Jaap1990** [comment]
https://github.com/openwebdocs/project/issues/42#issuecomment-879679064

**Comment by Elchi3** [comment]
We had a planning check-in call yesterday and here's a summary of where we are with Q3 projects:

## WIP with the goal to finish

- [Markdown conversation for more of MDN (CSS docs)](https://github.com/openwebdocs/project/issues/25) (Will)
- [WebXR Browser Compatibility](https://github.com/openwebdocs/project/issues/35) (Florian)
- [Documenting HTTP/2 and HTTP/3](https://github.com/openwebdocs/project/issues/43) (Jean-Yves)

## Aiming to make progress (at least)

- [Mixin work (for workers, events, etc.)](https://github.com/openwebdocs/project/issues/23) (help to unstuck, Florian)
- [Fix how MDN distributes Polyfills](https://github.com/openwebdocs/project/issues/27) (start discussions/ make plan that can be executed in Q4, Jean-Yves)
-  [Use webref as the source for CSS formal syntax, instead of mdn/data](https://github.com/openwebdocs/project/issues/44) (exploration, Will)

**Comment by foolip** [comment]
@Elchi3 do you consider [representing support of events](https://github.com/mdn/browser-compat-data/issues/7545) in score for Q3, or just the bits that get entangled with mixins? From my work on API data with @vinyldarkscratch, I would say that events are the most problematic part of the data, and while the model we have makes it possible to represent support for different facets of events, I don't think the result is very clear to web developers.

Spending some cycles on events would be great, if OWD could :)

**Comment by foolip** [comment]
Predicting that the answer is "no not yet" my follow-up question would be where I can park the suggestion for Q4 review.

**Comment by Elchi3** [comment]
I'm still busy with WebXR docs, but unblocking mixins when I can is in scope. The top of my to-list includes https://github.com/mdn/browser-compat-data/pull/11518. Let me know what other things are pressingly needed and I will try to get to it.

https://github.com/mdn/browser-compat-data/issues/7545 looks like a larger effort, though (and maybe it is separate enough from mixins?). So, it would be really great if you could file that as new project for OWD to consider for Q4. You can file an issue on this repo and we can flesh it out this quarter, so it is ready to be considered in the next planning call (September 8).
